examples/toggle.rs

```diff
@@ -0,0 +1,42 @@
+//! Uses the StatefulOutputPin embedded_hal trait to toggle the pin
+//! On the stm32 discovery board this is the red led
+//! Target board: STM32F3DISCOVERY
+
+#![deny(unsafe_code)]
+#![no_main]
+#![no_std]
+
+use panic_semihosting as _;
+
+use cortex_m_rt::entry;
+use stm32f3xx_hal::hal::digital::v2::OutputPin;
+use stm32f3xx_hal::prelude::*;
+use stm32f3xx_hal::stm32;
+
+#[entry]
+fn main() -> ! {
+    let p = stm32::Peripherals::take().unwrap();
+
+    let mut rcc = p.RCC.constrain();
+    let mut gpioe = p.GPIOE.split(&mut rcc.ahb);
+
+    let mut led_red = gpioe
```

Comment:
Might we be able to name this just `led`? The LED on my board is white :stuck_out_tongue_closed_eyes:

Reply:
Oh yeah right :grinning: I've not considered, that the led colors are not consistent between the discovery boards.